support-privacy.md

```diff
@@ -30,12 +30,18 @@ When you request a feature key, we may gather:
 * Details about you and your and your organization to help us determine the best way to support you.
 * Any other information listed on the sign-up form.
 
-This information is used to deliver the service to you. The billing information will not be shared with anyone. We assume the remaining information is non-secret and we may share it with business partners unless you tell us otherwise.
+This information is used to deliver the service to you. The billing information will be shared only with our payment processor (Stripe). We assume the remaining information is non-secret and we may share it with business partners unless you tell us otherwise by e-mailing [support@sandstorm.io](mailto:support@sandstorm.io).
+
+## Data gathered during free trials
+
+If you request a free trial, during the trial period your Sandstorm server will automatically share some statistics with us, such as the number of users on your server and the number of people using each app. To review the statistics being shared, visit the "statistics" page in the admin panel of your server (i.e. `<your-server>/admin/stats`). This information is only required to be shared during trials. Once you have purchased a non-trial key, you may choose whether or not you wish to continue to share this information.
```

Comment:
> Once you have purchased a non-trial key, you may choose whether or not you wish to continue to share this information.

A free feature key also passes this check, yeah? If so, it might be worth spelling that out.